src/components/ItemList.svelte

```diff
@@ -0,0 +1,59 @@
+<script>
+  import { _ } from "lodash";
```

Comment:
@wlach Using named import here to bypass the snapshot test, but the story would not render. Only default import seems to work in storybook, but if I use default import the test fails. Not sure what to do here

Reply:
Changing this to `import { capitalize } from "lodash"` seems to fix it!